R/pkg/R/DataFrame.R

```diff
@@ -677,13 +677,15 @@ setMethod("unique",
 #' collect(sample(df, TRUE, 0.5))
 #'}
 setMethod("sample",
-          # TODO : Figure out how to send integer as java.lang.Long to JVM so
```

Comment:
don't delete this comment. move it close to as.integer(seed). This is a known limitation of serde now.